### PR TITLE
Fix unet loading issue

### PR DESCRIPTION
**Pull request by AbigailMcGovern**
Changes:

- Upload newest version of unet
- changed options.package data in setup.cfg to include unet in distribution

### CODE REVIEW

**Comment by AbigailMcGovern** [comment]
ping @jni

File: setup.cfg [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [metadata]
 name = iterseg
-version = 0.0.8
+version = 0.0.10
```
Comment (jni) [inline setup.cfg]: LOL Maybe it's time to call it 0.1.0? 😂

File: setup.cfg [hunk 2]
```diff
@@ -28,6 +28,7 @@ project_urls =
 
 [options]
 packages = find:
+include_package_data = True
```
Comment (jni) [inline setup.cfg:31]: This one is actually unnecessary because you had it a few lines below, under the `install_requires`. So you can remove it.

Comment (jni) [reply setup.cfg:31]: (Or remove the bottom one, up to you.)

**Comment by jni** [comment]
Gah, still getting the error. For reference here's what I'm doing to test it:

- build this package with `python -m build .`
- install the built package (not in-place) with `pip install dist/iterseg-0.0.10-py3-none-any.whl`
- launch napari with `napari --plugin napari-ome-zarr path/to/platelets-image.zarr`
- launch iterseg / segment platelets
- click run

Also .whl's are just zip files so after the build step you can just `cd dist; unzip iterseg-0.0.10-py3-none-any.whl` and see whether the unet is there...

Got a meeting now but will look again later... I don't know off the top of my head why it's still not getting packaged.

**Comment by jni** [comment]
Ok this commit seems informative 🤣 

https://github.com/jni/platelet-unet-watershed/commit/66aab7620d2e1646e3d4d33fe85c1691c63353cb

tldr make sure you have a MANIFEST.in file and include the `data` directory there.

You are probably wondering why you need to specify to include this thing in so many places, and you would be right to wonder that. 🤣 I have no earthly idea.

**Comment by jni** [comment]
(And make sure you include the LICENSE file as well.)

**Comment by jni** [comment]
btw this is reasonably urgent because the reviewers at some point will be testing the package and it should work! 😬

File: MANIFEST.in [hunk 3]
```diff
@@ -4,3 +4,5 @@ include requirements.txt
 
 recursive-exclude * __pycache__
 recursive-exclude * *.py[co]
+
+graft iterseg/data
```
Comment (jni) [inline MANIFEST.in]: try changing this to:

```
graft src/iterseg/data
```

?

Comment (jni) [reply MANIFEST.in]: ah, that includes all the .zarr files which I don't think we want to ship. Maybe:

```suggestion
graft src/iterseg/data/*.pt
```

Comment (jni) [reply MANIFEST.in]: ... still bringing in the zarrs. wth.

Comment (jni) [reply MANIFEST.in]: maybe move the zarrs (and csv) to a different dir, say, `_local_data/`?

Comment (jni) [reply MANIFEST.in]: ok, sorry for the wild goose chase — this seems to work:

```suggestion
recursive-include data *.pt
```

Still building but I see this line in the logs:

```
creating iterseg-0.1.0/data
```

which is encouraging!

Comment (jni) [reply MANIFEST.in]: gdi, no it did not include it!!!! 😡

Comment (jni) [reply MANIFEST.in]: Weird... The *source* distribution has it. But not the wheel. 🤔

**Comment by jni** [comment]
@AbigailMcGovern I don't want to merge this until the zarrs are gone. 😅 

btw am I to understand that your unet is numbered YYDDMM? Because that is crazy 😂
